ucfcs/GrowthPlus
# Julio/game points

This PR implements the pointing system for the roadmap games. 

To test:
Play roadmap game 1, 2, 3, 4 :
- Each correct answer should update the top bar by one point, you can earn a max of 20 points, and must earn 17 to pass, if completed, the UI  of the tile should update to completed state.
- You can still earn points if going back to play if current game score is less than 20. 
- Once you reached the max points, you can't earn any more points. 
-  Try going back to roadmap with the back button while on game, if score >= 17, the state of the tile should update to completed.
